docs/src/main/sphinx/security/certificate.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+You can configure Trino to support client-provided certificates validated by the
+Trino server on initial connection.
+
+.. important::
+
+    This authentication method is only provided to support sites that have an
+    absolute requirement for client authentication *and already have* client
+    certificates for each client. Sites in this category have an existing PKI
+    infrastructure, possibly including an onsite Certificate Authority (CA).
+
+    This feature is not appropriate for sites that need to generate a set of
+    client certificates in order to use this authentication type. Consider
+    instead using another :ref:`authentication type <cl-access-auth>`.
+
+Using certificate authentication
+--------------------------------
+
+All clients connecting with HTTPS/TLS go through the following initial steps:
+
+1. The client attempts to contact the coordinator.
+2. The coordinator returns its certificate to the client.
+3. The client validates the server's certificate using the client's trust store.
+
+A cluster with certificate authentication enabled goes through the following
+additional steps:
+
+4. The coordinator asks the client for its certificate.
+5. The client responds with its certificate.
+6. The coordinator verifies the client's certificate, using the coordinator's
+   trust store.
+
+Several rules emerge from these steps:
+
+* Trust stores used by clients must include the certificate of the signer of
+  the coordinator's certificate.
+* Trust stores used by coordinators must include the certificate of the signer
+  of client certificates.
+* The trust stores used by the coordinator and clients do not need to be the
+  same.
+* The certificate that verifies the coordinator does not need to be the same as
+  the certificate verifying clients.
+
+Trino validates certificates based on the distinguished name (DN) from the
```

Comment:
Remove the second sentence and change "CN" in the last sentence to "subject DN"

Reply:
Done!